srcpkgs/massdns/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'massdns'
+pkgname=massdns
+version=0.3
+revision=1
+build_style=gnu-makefile
+short_desc="High-performance DNS stub resolver for bulk lookups and reconnaissance"
+maintainer="Antoine Guilbaud <antoine.guilbaud@yahoo.fr>"
+license="GPL-3.0-only"
+homepage="https://github.com/blechschmidt/massdns"
+distfiles="https://github.com/blechschmidt/massdns/archive/v${version}.tar.gz"
+checksum=9ca6e684d2febf707f165ce74b7a38455302f5851309fb82b5951c28cb07d0d7
+
+post_install() {
+	vbin bin/massdns
```

Comment:
this should probably overwrite `do_install`, which is currently leaking the binaries into the build chroots `/usr/bin`.
https://travis-ci.org/github/void-linux/void-packages/jobs/676937078#L4145